src/java.base/share/classes/java/net/ServerSocket.java

```diff
@@ -184,8 +184,9 @@ public ServerSocket(int port) throws IOException {
      * connection is refused.
      * <p>
      * If the application has specified a server socket implementation
-     * factory, that factory's {@code createSocketImpl} method is called to
-     * create the actual socket implementation. Otherwise a system-default
+     * factory, that {@linkplain SocketImplFactory#createSocketImpl()
+     * factory's createSocketImpl method} is called to create
+     * the actual socket implementation. Otherwise a system-default
```

Comment:
I think this would be better if you link  "server socket implementation factory" to SocketImplFactory, and link "createSocketImpl" to the createSocketImpl method.